### PR TITLE
feat: WIP checking if mod is outdated

### PR DESCRIPTION
**Pull request by GeckoEidechse**
I wouldn't merge this tbh. Can be done a lot nicer, I just don't know how ^^"

### CODE REVIEW

File: src-vue/src/views/ThunderstoreModsView.vue [hunk 1]
```diff
@@ -145,9 +145,17 @@ export default defineComponent({
             if (this.modsBeingInstalled.includes(mod.name)) {
                 return ThunderstoreModStatus.BEING_INSTALLED;
             }
-            // TODO ensure mod is up-to-date
-            if (this.$store.state.installed_mods.map((mod: NorthstarMod) => mod.thunderstore_mod_string).includes(mod.versions[0].full_name)) {
-                return ThunderstoreModStatus.INSTALLED;
+            for (let installed_mod of this.$store.state.installed_mods) {
+                if (installed_mod.thunderstore_mod_string !== null) {
+                    if (mod.full_name == installed_mod.thunderstore_mod_string.slice(0, -6)) {
```
Comment (GeckoEidechse) [inline src-vue/src/views/ThunderstoreModsView.vue:150]: This will break the moment a mod with multi digit version numbers is checked, e.g. `1.2.69`

Comment (Alystrasz) [reply src-vue/src/views/ThunderstoreModsView.vue:150]: I've done it a bit differently, see https://github.com/GeckoEidechse/FlightCore/pull/54/commits/87f95372d48947ccffbdf78964a055adde6553a9 for details

Comment (GeckoEidechse) [reply src-vue/src/views/ThunderstoreModsView.vue:150]: Yeah, that's an actually viable implementation :P

**Comment by GeckoEidechse** [comment]
Closing as implemented with 87f95372d48947ccffbdf78964a055adde6553a9

**Pull request closed without merging**